internal/provisioner/plain/README.md

```diff
@@ -1,8 +1,8 @@
-# Plain-v0 Provisioner
+# Plain Provisioner
 
 ## Summary
 
-The `plain-v0` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
+The `plain` provisioner is a core rukpak provisioner that knows how to interact with `plain+v0` format bundles.
```

Comment:
Maybe it is worth pointing out very explicitly that `plain` is the Provisioner and `plain+v0` is just the version of the format it uses.

Reply:
It probably is extremely confusing for newer users now that I'm thinking about it...as someone looking at this for a while it seems normal to me.

Reply:
Especially right at the beginning of the README, might be off-putting. I will update.